crates/rbuilder/src/building/order_commit.rs

```diff
@@ -1012,15 +1012,8 @@ impl<'a, 'b, Tracer: SimulationTracer> PartialBlockFork<'a, 'b, Tracer> {
                 match res {
                     Ok(ok) => {
                         let coinbase_balance_after = self.state.balance(ctx.block_env.coinbase)?;
-                        let coinbase_profit = match coinbase_profit(
-                            coinbase_balance_before,
-                            coinbase_balance_after,
-                        ) {
-                            Ok(profit) => profit,
-                            Err(err) => {
-                                return Ok(Err(err));
-                            }
-                        };
+                        let coinbase_profit =
```

Comment:
Can you please add comment explaining why we don't check it for txs and bundle? Its basically because builder does not sign txs in this code path.